dev-docker/docker-compose.yml

```diff
@@ -33,6 +33,10 @@ services:
     environment:
       DJANGO_SETTINGS_MODULE: weblate.settings_docker
       WEBLATE_SITE_DOMAIN: ${WEBLATE_HOST}
+      WEBLATE_GITHUB_USERNAME: qwerty287
+      WEBLATE_GITHUB_TOKEN: 95a2a068275ced3452db6d57aed5fa0ade82a50f
+      WEBLATE_GITEA_USERNAME: qwerty287
+      WEBLATE_GITEA_TOKEN: 95a2a068275ced3452db6d57aed5fa0ade82a50f
```

Comment:
If you need some changes, add them to the override file, not here.

Reply:
Sorry, this was just for testing purposes and I forgot to remove it.